CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+`1.5.1`_ - Not yet released
```

Comment:
`1.5.1`_ is actually a hyperlink. Look at line 222 in the file :smile:   

In short, you need to add this: 
```rst
.. _1.5.1: https://github.com/snok/django-auth-adfs/compare/1.5.0...1.5.1
```

You can also just change the date to the date you've resolved these comments and I'll merge it that day.

Reply:
Thanks -- RST is not my main doco language (it's mostly markdown). Fixed.